packages/disploy/cli/src/commands/dev.ts

```diff
@@ -6,48 +6,71 @@ import ora from 'ora';
 import type { Argv, CommandModule } from 'yargs';
 import { createServer, setApp } from '../lib/devServer';
 import { ProjectTools } from '../lib/ProjectTools';
+import { runShellCommand } from '../lib/shell';
 import { F } from '../lib/StringFormatters';
 import { logger } from '../utils/logger';
 import { BuildApp } from './common/build';
 
 export const aliases: string[] = [];
 export const desc: string = 'Enter development mode';
 
-export const builder = (yargs: Argv) => yargs.options({});
+export const builder = (yargs: Argv) =>
+	yargs.options('ignore-watcher-output', {
+		alias: 'iwo',
+		default: false,
+		type: 'boolean',
+	});
 
-export const DevCommand: CommandModule = {
+export const DevCommand: CommandModule<{}, { 'ignore-watcher-output': boolean }> = {
 	aliases,
 	builder,
 	command: 'dev',
-	async handler() {
+	async handler(opts) {
 		const devServerPort = 5002;
 
-		const { root } = await ProjectTools.resolveProject({
+		const {
+			root,
+			prebuild,
+			watcher: devScript,
```

Comment:
why the need to rename to devScript?

Reply:
The fs watcher is also called `watcher`.